stories/core/Modal.stories.tsx

```diff
@@ -90,7 +90,7 @@ export const Basic: Story<ModalProps> = ({
         onKeyDown={action('onKeyDown', { depth: 1 })}
         {...rest}
       >
-        <div>
+        <div className='iui-modal-content'>
```

Comment:
Maybe content needs to have its own component? Like `ModalButtonBar`. Then users won't need to add `iui-modal-content` manually to their code.